src/uwtools/drivers/stager.py

```diff
@@ -0,0 +1,56 @@
+"""
+A base class for tasks that stage files.
+"""
+
+from iotaa import tasks
+
+from uwtools.fs import Copier, Linker
+from uwtools.strings import STR
+
+
+# mypy: disable-error-code="attr-defined"
+class FileStager:
+    """
+    A base class for tasks that stage files.
+    """
+
+    @tasks
+    def files_copied(self):
+        """
+        Files copied for run.
+        """
+        yield self.taskname("files copied")
+        yield [
+            Copier(
+                config=self.config.get("files_to_copy", {}),
+                target_dir=self.rundir,
+            ).go()
+        ]
+
+    @tasks
+    def files_hardlinked(self):
+        """
+        Files hard-linked for run.
```

Comment:
```suggestion
        Files hardlinked for run.
```
Can I propose that we treat "hardlink" as a single word the same way we treat "symlink", for consistency and to avoid additional punctuation? What do you think? (Fixing it here and re-running `make docs` should update all the occurrences in the RST docs.)